packages/utilities/psammead-styles/src/index.stories.jsx

```diff
@@ -51,9 +54,47 @@ const Detects = styled.li`
   }
 `;
 
+const Paragraph = styled.p`
+  ${Object.values(fonts).join()}
+`;
+
+const camelCase = str => str.replace(/-([a-z])/g, g => g[1].toUpperCase());
+const fontNames = Object.keys(fonts).sort();
+const fontStyles = fontNames.map(x => x.substring(2).replace(/_/g, ' '));
+
+const getFontStyles = fontName => {
+  const font = fontNames.find(x => x.includes(fontName.replace(/ /g, '_')));
+  const fontFace = fonts[font] || '';
+  const fontStyle = fontFace
+    .replace(/"/g, '')
+    .replace('@font-face', '')
+    .replace(/: /g, ':"')
+    .replace(/;/g, '",');
+
+  const { fontFamily, fontWeight } = json5.parse(camelCase(fontStyle));
```

Comment:
Why do you need `json5` here? Why doesn't `JSON.parse` work? 🤔

Reply:
JSON.parse does not work for unquoted values. I need the extended functionality of [Json5](https://json5.org/)

Reply:
Ahhhh ok, for others benefit `camelCase(fontStyle)` is 
```js
{
  fontFamily:"Iskoola Pota BBC",
  fontWeight:"700",
  fontStyle:"normal",
  src:"url('https://wsDownloads.files.bbci.co.uk/fonts/IskoolaPota/v5.91/bold.woff') format('woff'), url('https://wsDownloads.files.bbci.co.uk/fonts/IskoolaPota/v5.91/bold.eot') format('eot'), url('https://wsDownloads.files.bbci.co.uk/fonts/IskoolaPota/v5.91/bold.ttf') format('ttf')",
  fontDisplay:"swap",
}
```